cmd/proxy/actions/filter.go

```diff
@@ -7,11 +7,24 @@ import (
 
 	"github.com/bketelsen/buffet"
 	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/config/env"
 	"github.com/gomods/athens/pkg/errors"
 	"github.com/gomods/athens/pkg/module"
 	"github.com/gomods/athens/pkg/paths"
 )
 
+const (
+	// OlympusGlobalEndpoint is a default olympus DNS address
+	OlympusGlobalEndpoint = "http://localhost:3001"
```

Comment:
Is this file changed by mistake? this seems out of scope (and already addressed in another PR if im not mistaken)